ajax/libs/dependent-dropdown/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "dependent-dropdown",
+  "filename": "js/dependent-dropdown.min.js",
+  "version": "1.4.8",
+  "author": "Kartik Visweswaran <kartikv2@gmail.com>",
+  "homepage": "http://plugins.krajee.com/dependent-dropdown",
+  "description": "A multi level dependent dropdown JQuery plugin that allows nested dependencies.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kartik-v/dependent-dropdown.git"
+  },
+  "keywords": [
+    "jquery",
+    "dependent",
+    "dropdown",
+    "select"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/kartik-v/dependent-dropdown.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "+(css|js)/*dependent-dropdown*",
+          "js/locales/(!LANG)*",
```

Comment:
@sashberd Would you try this package from npm? https://github.com/isaacs/node-glob#glob-primer (`npm i glob`)
This is a tool that you can know that how to get the files depend on the pattern.

When you work in terminal, you can enter into `node` environment after you `git clone dependent-dropdown` and change directory to `dependent-dropdown`.
Here are the demo:
```js
$ node
> glob = require('glob')
{ [Function: glob]
  sync: { [Function: globSync] GlobSync: [Function: GlobSync] },
  GlobSync: [Function: GlobSync],
  glob: [Circular],
  hasMagic: [Function],
  Glob:
   { [Function: Glob]
     super_:
      { [Function: EventEmitter]
        EventEmitter: [Circular],
        usingDomains: true,
        defaultMaxListeners: 10,
        init: [Function],
        listenerCount: [Function] } } }
> glob.sync('js/locales/(!LANG)*')
[]
> glob.sync('js/**/*')
[ 'js/dependent-dropdown.js',
  'js/dependent-dropdown.min.js',
  'js/locales',
  'js/locales/de.js',
  'js/locales/es.js',
  'js/locales/fr.js',
  'js/locales/it.js',
  'js/locales/LANG.js',
  'js/locales/pl.js',
  'js/locales/pt.js',
  'js/locales/ru.js' ]
```

Would you try to fix this?
I think we just need to exclude the `js/locales/LANG.js',` :)

Thanks

Reply:
@sufuf3 I used this one in order to check my patterns